src/components/home/SidePanel.vue

```diff
@@ -26,7 +26,7 @@ const leftAlignment = computed(() =>
 <template>
   <div class="flex shadow-sm min-h-screen">
     <div
-      class="px-6 py-8 bg-white flex flex-col gap-2 w-[450px] max-h-screen overflow-y-auto scrollbar-hide"
+      class="px-6 py-8 bg-white flex flex-col gap-2 space-y-4 w-[450px] max-h-screen overflow-y-auto scrollbar-hide"
```

Comment:
what is the difference in adding `space-y-4` while we already have `gap-2`?
just my curiosity